guides/reference/windows.md

```diff
@@ -1 +1,83 @@
-<!-- Port over info from existing site -->
\ No newline at end of file
+Windows Vista and newer windows versions are fully supported.
+
+To get started ensure the following dependencies are installed:
+
+* Node.js - [https://nodejs.org/en/](https://nodejs.org/en/)
+* Git - [https://git-scm.com/](https://git-scm.com/)
+* Phantom.js - [http://phantomjs.org/](http://phantomjs.org/)
+
+### Performance
+
+Although supported, Windows performance, at least by default, isn't as good as
+on Linux or MacOS. On a positive note, this story continues to improve. Both
+Microsoft, and the Ember CLI team continue to work to improve these developer
```

Comment:
Could also write `continue to work on improving these…`

Reply:
updated.